datasets/kz/kz_companies/kz_companies.yml

```
@@ -0,0 +1,28 @@
+name: kz_companies
+title: State Register of legal entities in the Republic of Kazakhstan
+prefix: kz
+disabled: false
+entry_point: crawler
+coverage:
+  frequency: weekly
+  start: 2023-03-21
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+deploy:
+  # premium: true
+resolve: false
+ci_test: false
+http:
+  total_retries: 5
+  backoff_factor: 3
+  retry_statuses: [503, 429, 403]
+
+summary: |
+  All companies in Kazakhstan, with information on their directors.
+url: https://data.egov.kz/datasets/view?index=gbd_ul
+publisher:
+  name: Ministry of Justice of the RK
+  country: kz
+  url: https://www.gov.kz/memleket/entities/adilet?lang=en
+  description: |
+    The Ministry of Justice of the Republic of Kazakhstan is a holder of the company registry.
```

I would make this a bit more general about the publisher using the snippet from their about page
```suggestion
  description: |
    The Ministry provides legal support for the activities of the state, supports the rule of law in the work of state bodies, organizations, officials and citizens, provides protection of the rights and legal interests of citizens and organizations.
```